github-usercard/src/components/UserCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class UserCard extends React.Component{
+    render() {
+        return (
+            <div>
+                {this.props.name}
```

Comment:
The pieces here are erroring out for me for some reason.  I don't know why.  I'll have to have you show me the app